docs/architecture/microservices/net-core-net-framework-containers/official-net-docker-images.md

```diff
@@ -8,7 +8,7 @@ ms.date: 01/07/2019
 
 The Official .NET Docker images are Docker images created and optimized by Microsoft. They are publicly available in the Microsoft repositories on [Docker Hub](https://hub.docker.com/u/microsoft/). Each repository can contain multiple images, depending on .NET versions, and depending on the OS and versions (Linux Debian, Linux Alpine, Windows Nano Server, Windows Server Core, etc.).
 
-Since .NET Core 2.1, all the .NET Core images, including for ASP.NET Core are available at Docker Hub at the .NET Core image repo: https://hub.docker.com/_/microsoft-dotnet-core/
+Since .NET Core 2.1, all the .NET Core images, including for ASP.NET Core are available at Docker Hub at the .NET Core image repo: <https://hub.docker.com/_/microsoft-dotnet-core/>
```

Comment:
I think you need to escape the _ 

```suggestion
Since .NET Core 2.1, all the .NET Core images, including for ASP.NET Core are available at Docker Hub at the .NET Core image repository: <https://hub.docker.com/\_/microsoft-dotnet-core/>.
```